docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md

```diff
@@ -111,3 +95,11 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 }
 
 ```
+
+### Terraform example
```

Comment:
@JV0812, there is no Terraform example under this heading. Should there be one?

Reply:
Yes John, its a placeholder to add the Terraform example. I have requested the same from SME, if he provides then I will add that here.

Reply:
@JV0812, if you are waiting on Terraform examples, should this PR be in draft instead of "Ready to review"?